content/docs/start/experiments-trail/experiments-1.md

```diff
@@ -0,0 +1,334 @@
+run the commands in this document after cloning the repository and installing
+the requirements.
+
+### Clone the project and create virtual environment
+
+Please clone the project and create a virtual environment.
+
+> We create a virtual environment to keep the libraries we use isolated from the
+> rest of your system. This prevents version conflicts.
+
+```dvc
+$ git clone https://github.com/iterative/get-started-experiments -b source-code
+$ cd get-started-experiments
+$ virtualenv .venv
+$ . .venv/bin/activate
+$ python -m pip install -r requirements.txt
+```
+
+### Get the data set
+
+The repository you cloned doesn't contain the dataset. In order to get
+`fashion-mnist.tar.gz` from `dataset-registry`, we use `dvc get` to download the
+missing data files. `dvc get` is similar to `wget` but works with Git+DVC
+repositories to download binary files.
+
+```dvc
+$ dvc get https://github.com/iterative/dataset-registry \
+          fashion-mnist/images.tar.gz -o data/images.tar.gz
+```
+
+Then we extract this file that contains labeled images.
+
+```dvc
+$ tar -xvzf data/images.tar.gz --directory data/
+```
+
+### Specify the experiment
+
+We first initialize DVC inside the project to create an experiment.
+
+```dvc
+$ dvc init
+```
+
+Then we add the dataset to the project:
+
+```dvc
+$ dvc add data/images
+```
+
+This creates a `data/images.dvc` file that contains all the relevant metadata of
+the directory. You can add and commit `data/images.dvc` file, and changes in
+`.gitignore` that hides the `data/images/` directory from Git.
+
+```dvc
+$ git add data/.gitignore data/images.dvc
+$ git commit -m "Dataset added"
+```
+
+DVC experiments are run by specifying their commands, outputs, parameters and
+dependencies. We specify an experiment command by `dvc stage add`.
+
+```dvc
+$ dvc stage add -n train \
+                -p model.conv_units \
+                -p train.epochs \
+                -d data/images \
+                -m metrics.json \
+                python3 src/train.py
+```
+
+The command tells DVC to create an experiment named `train`, and that for any
+change in `data/images/`, `model.conv_units` or `train.epochs`, we run an
+experiment using `src/train.py` that produces a new `metrics.json` file.
+
+DVC creates `dvc.yaml` file and modifies `.gitignore` for Git to ignore certain
+artifacts.
+
+```dvc
+$ git add dvc.yaml .gitignore
+$ git commit -m "added data and the experiment"
+```
+
+DVC is ready to run the experiments now!
+
+</details>
+
+## Running the experiment with default parameters
+
+The purpose of `dvc exp` subcommands is to run the pipeline for ephemeral
+experiments. By _ephemeral_ we mean the experiments can be run without
+committing parameter and dependency changes to Git. Instead the artifacts
+produced for each experiment are tracked by DVC and persisted on demand.
+
+Running the experiment with default hyperparameter values requires only the
+command:
+
+```dvc
+$ dvc exp run
```

Comment:
~~`default hyperparameter values`:~~
- ~~What do we mean by "default" here? Sounds like DVC auto-fills them.~~
- ~~Link to https://dvc.org/doc/command-reference/params ?~~

Reply:
Actually, maybe no need to mention params yet since the next section focuses on that? Instead, this section could be more general and talk about data and code changes, or even show basic ones.

Reply:
default hyperparameter values ➡ default project settings

Reply:
Yes but what is "default" here? The reader may not have context as to the specific contents of the sample project if they're just reading casually. Even if they're following but don't have DVC experience they won't know what the default project settings or params are or where to find them. Maybe there's another more general way to say this, like "Use `dvc exp run` to run an experiment based on the current DVC metafiles".

Reply:
It's still not possible to run an experiment without pipeline, so I think the user is expected to have some "default values" or "default pipeline" defined in their project.